sync2jira/downstream_issue.py

```diff
@@ -1034,7 +1034,7 @@ def _update_assignee(client, existing, issue, updates):
                 confluence_data = {'Assignee': 1}
                 confluence_client.update_stat_page(confluence_data)
         else:
-            if existing.fields.assignee and existing.fields.assignee.displayName != issue.assignee[0]['fullname']:
+            if existing.fields.assignee and not issue.assignee:
```

Comment:
No one uses assignee so that why it wasn't caught, but I noted this did not make sense so I fixed it in this PR as well.

Reply:
Nice.  :)